packages/dashboard-frontend/src/services/dashboard-backend-client/devWorkspaceApi.ts

```diff
@@ -112,3 +113,14 @@ export async function injectKubeConfig(namespace: string, devworkspaceId: string
     throw `Failed to inject kubeconfig. ${helpers.errors.getMessage(e)}`;
   }
 }
+
+export async function getDevfileSchema(
+  schemaVersion: string,
+): Promise<JSONSchema7 | { [key: string]: any }> {
```

Comment:
Is it possible to avoid using the type `any` here?

```suggestion
): Promise<JSONSchema7 | { [key: string]: unknown }> {
```

Reply:
Ok. I have fixed it.